### ISSUE TITLE
[Bug] [Android] Result photo is different from actual camera preview screen

### ISSUE THREAD

**Issue by thecuong064** [xamarin/XamarinCommunityToolkit#1612]
### Description
Capturing photo with CameraView covering the whole screen. The result photo will be different from actual camera preview screen on Android.

I have just tested on Android so there's no idea whether this happens on iOS.

### Steps to Reproduce
- Use the attached project below

### Expected Behavior
The result photo is exactly the same as camera preview.

### Actual Behavior
The result photo is different from camera preview.
.
### Basic Information

- Version with issue: 1.2.0
- Last known good version: Unknown
- Affected Devices: Samsung Galaxy S6

### Reproduction sample
[CustomCameraDemo.zip](https://github.com/xamarin/XamarinCommunityToolkit/files/7126147/CustomCameraDemo.zip)

### Reproduction imagery
<img src="https://user-images.githubusercontent.com/23501058/132452281-49935462-9759-468e-b9ee-cdaa975089e1.png" width="180" height="320">

**Comment by GUOLDEV** [comment]
@thecuong064 how are you comparing the images? Based on the "Reproduction imagery"?

**Comment by thecuong064** [comment]
Hi @GUOLDEV,

As you can see the camera preview and the image preview is not the same. This is the screen after I finished taking a photo. Let me highlight the image preview for you.
![image](https://user-images.githubusercontent.com/23501058/133010923-78b893a9-ccb1-48c6-ba50-0e4d530639d9.png)

**Comment by GUOLDEV** [comment]
Hi @thecuong064, the preview image is not the same proportion as the screen, and you are probably using `Aspect="AspectFit"`, so the image zooms to fit the control size. Does that make sense?

**Comment by thecuong064** [comment]
I'm using that AspectFit property actually. You can get more details from my above attached project.
![image](https://user-images.githubusercontent.com/23501058/133024120-16ba97d7-c3c2-4c86-bd5a-bee242f0a506.png)

**Comment by GUOLDEV** [comment]
So, AspectFit is going to change the size of the image to fit it to the size of the control, so it is not going to look exactly like the real image. For example, if you set the HeightRequest and WidthRequest as 100, it is going to show a squared image, cropping part of the image, the top, and button. Even the default camera app in the device does not show exactly the same image on the preview, but I get your point. You are probably right about this issue though, but for me, the image looks just slightly different.

**Comment by thecuong064** [comment]
Yeah, it's slightly different. But I just want to correct a litte bit about the AspectFit, referring to this [link ](https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.aspect?view=xamarin-forms)from Microsoft

 **AspectFit does not clip/crop** any parts of the image. It will left the space empty if the width or height does not fit enough to the container and showing the full image with correct aspect ratio.

**AspectFill** will crop/clip some parts of the image.
![image](https://user-images.githubusercontent.com/23501058/133033397-a5347299-bf46-4829-9747-a39a5906de0d.png)

In my case, I didn't declare any `WidthRequest `. Image `Width` will be fit, based on the ratio, with the `HeightRequest = 100`.

**Comment by huyvuskedulo** [comment]
Let me explained the problem a little bit more.

The Aspect mode which CameraView is using is actually like AspectFill in Image view, so the Camera Preview View is actually cropping the the actual image it will take to fit the given view port, which is not always the desired result. Instead, the CameraView should have a Aspect property like AspectFill and AspectFit and let the developer decide if the Preview view should take the whole view port or respect the actual result coming from the Camera.
